cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/images/AddLaunchPermissionsToImageAction.java

```diff
@@ -104,8 +105,8 @@
                                        @Param(value = USER_IDS_STRING) String userIdsString,
                                        @Param(value = USER_GROUPS_STRING) String userGroupsString) {
         try {
-            version = InputsUtil.getDefaultStringInput(version, "2016-04-01");
-            CommonInputs inputs = new CommonInputs.Builder()
+            version = InputsUtil.getDefaultStringInput(version, API_VERSION);
```

Comment:
I suggest to use the "hardcoded" value for version since these were the last versions we tested on and if for example we want to update only 1 operation we will have to update all operations and maybe other operations will not support that version